test/test_automl/test_fit_ensemble.py

```diff
@@ -1,35 +0,0 @@
-import numpy as np
-
-from autosklearn.automl import AutoML
-
-import pytest
-from pytest_cases import filters as ft
-from pytest_cases import parametrize, parametrize_with_cases
-
-import test.test_automl.cases as cases
-
-
-@parametrize("ensemble_size", [-10, -1, 0])
-@parametrize_with_cases("automl", cases=cases, filter=~ft.has_tag("fitted"))
-def test_non_positive_ensemble_size_raises(
-    tmp_dir: str,
-    automl: AutoML,
-    ensemble_size: int,
-) -> None:
-    """
-    Parameters
-    ----------
-    automl: AutoML
-        The AutoML object to test
-
-    ensemble_size : int
-        The ensemble size to use
-
-    Expects
-    -------
-    * Can't fit ensemble with non-positive ensemble size
-    """
-    dummy_data = np.array([1, 1, 1])
-
-    with pytest.raises(ValueError):
-        automl.fit_ensemble(dummy_data, ensemble_size=ensemble_size)
```

Comment:
Should probably replaced by it's new counterpart behaviour in that no `ensemble_class` will raise.

Reply:
I can do so, but I'm not 100% if it makes a lot of sense. The user would have to set `ensemble_class=None` in both the constructor and `fit_ensemble`.